### DISCUSSION TITLE
The task icon doesn't like others.

### DISCUSSION THREAD

**Discussion by dapeng901** [apache/dolphinscheduler#10743] [General]
My task icon is like in the following imge 
![image](https://user-images.githubusercontent.com/9891296/177022472-ec2cb69c-b8ac-417e-8095-63c872a4738e.png)

the official task icons are as the belows. How can i change my icons into the same as the officals
![image](https://user-images.githubusercontent.com/9891296/177022510-272af40b-c875-4668-b819-83116ece9016.png)

**Comment by vivostar** [comment]
The first picture is new UI v2(vue3),  the second picture is old edtion UI v1(vue2).
